scripts/setup.sh

```
@@ -91,30 +111,30 @@ fi
+rm -rf .sync-env
 python -m venv .sync-env
 source .sync-env/bin/activate
 pip install --upgrade pip
 pip install -r requirements.txt
 deactivate
-cd ..
 
 # ---- Set up the frontend ----
 echo -e "${YELLOW}Setting up frontend...${NC}"
-cd frontend
+cd "$REPO_ROOT/frontend"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)